translation/fr-FR.toml

```diff
@@ -28,7 +28,7 @@ js-enter_password = "Entrez votre mot de passe pour supprimer votre compte DÉFI
 [index]
 secondary_title = "Le site officiel pour jouer en direct !"
 what_is_it_title = "Qu'est ce que c'est ?"
-what_is_it_pargaraphs = [
+what_is_it_paragraphs = [
```

Comment:
Changes to the TOML files need to be specified in translations/ changes.json. But it's okay if you didn't know that